CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 ### Added
 - `bin/upgrade` displays any changes to the changelog and prompts for
    confirmation before applying the remote changes to the local branch.
+   
+### Changed
+- Breaking change!!! LDAP and SAML features are now activated by two new
```

Comment:
```suggestion
- **Breaking change**: LDAP and SAML features are now activated by two new
```
WDYT about presenting this in bold instead of using multiple `!`?